webapp/src/modules/storage/selectors.js

```diff
@@ -1,2 +1,2 @@
 export const getState = state => state.storage
-export const isLoading = state => getState(state).loading
+export const isLoading = state => getState(state).loading >= 2
```

Comment:
I can't tell how many storage middlewares are there without doing some weird state management. That leaves this hardcoded `2` here, which actually means `amount of keys saved`.

I think this is far from optimal, we'll need to remember this next time we're changing the stored state, but I can't figure out a good way to do it